web/src/components/CodeSample/CodeSample.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react'
+import hljs from 'highlight.js'
+import hljsDefineGraphQL from 'highlightjs-graphql'
+hljsDefineGraphQL(hljs)
+const md = require('markdown-it')({
+  html: true,
+  linkify: true,
+  highlight: function (str, lang) {
+    if (lang && hljs.getLanguage(lang)) {
+      try {
+        return hljs.highlight(lang, str).value
+      } catch (__) {}
+    }
+
+    return '' // use external default escaping
+  },
+})
+
+const CodeSample = ({ provider }) => {
+  const [markdownCode, setMarkdownCode] = useState()
+
+  useEffect(() => {
+    async function getMarkdown() {
+      try {
+        const file = await import(`../../lib/code-samples/${provider.slug}.md`)
+        setMarkdownCode(md.render(file.default))
+      } catch {
+        setMarkdownCode(null)
+      }
+    }
+    getMarkdown()
+  })
+
+  return (
+    <>
+      {markdownCode ? (
+        <div
+          className="markdown mt-6"
+          dangerouslySetInnerHTML={{ __html: markdownCode }}
```

Comment:
With remark you don't have to use `dangerouslySetInnerHTML`

Reply:
Resolving since decided not to use remark for moment.